backend/src/main/java/com/bakdata/conquery/io/jackson/serializer/MetaIdRef.java

```diff
@@ -13,12 +13,12 @@
 
 /**
  * An annotation that guides Jackson to serialize/deserialize the field as a {@link NamespacedId} instead of the object content itself.
- *
  */
 @Retention(RetentionPolicy.RUNTIME)
 @JacksonAnnotationsInside
 @JsonProperty
-@JsonSerialize(using=IdReferenceSerializer.class)
-@JsonDeserialize(using=MetaIdReferenceDeserializer.class)
+@JsonSerialize(using = IdReferenceSerializer.class)
+@JsonDeserialize(using = MetaIdReferenceDeserializer.class)
 @Target({ElementType.FIELD, ElementType.TYPE_PARAMETER, ElementType.TYPE_USE})
-public @interface MetaIdRef {}
\ No newline at end of file
+public @interface MetaIdRef {
```

Comment:
Irgendwie ist das an mir vorbeigegangen, dass die raus war

Reply:
Das Problem mit der MetaIdRef ist, dass sie nicht wirklich funktioniert innerhalb des Clusters. Da nur der Manager den MetaStorage hat. Dh sie ist nur in der API nützlich wo wir sie nicht benutzen.